test_lib/cql_types.py

```diff
@@ -0,0 +1,104 @@
+import random
+
+
+ALL_TYPES = ('ascii', 'blob', 'counter', 'decimal', 'duration', 'int', 'map', 'smallint', 'time',
+             'timeuuid', 'uuid', 'varint', 'bigint', 'boolean', 'date', 'double', 'float', 'inet', 'list', 'set',
+             'text', 'timestamp', 'tinyint', 'varchar')
+NOT_EMBEDDABLE_TYPES = ['duration', 'counter']
+FROZEN_EMBEDDABLE_TYPES = ['set', 'list', 'map']
```

Comment:
`FROZEN_EMBEDDABLE_TYPES` -> `COLLECTION_TYPES`
frozen has a different meaning in Scylla and is misleading